### PR TITLE
Fiber shift

### PR DESCRIPTION
**Pull request by syhaffert**
Added a keyword to the StepIndexFiber so that it can be placed at other positions than the origin.

### CODE REVIEW

File: hcipy/optics/fiber.py [hunk 1]
```diff
@@ -64,6 +66,17 @@ def core_radius(self, core_radius):
 		self._core_radius = core_radius
 		self.clear_cache()
 
+	@property
+	def position(self):
```
Comment (ehpor) [inline hcipy/optics/fiber.py:70]: Why mix `position` and `center`? I'd rather choose one or the other. My preference goes to `position`, but feel free to let me know any reason for the other name.

**Review by ehpor — CHANGES_REQUESTED** [review]

File: hcipy/optics/fiber.py (hunk 1 above)
Comment (syhaffert) [reply hcipy/optics/fiber.py:70]: I have adjusted the name. There was no particular reason why they were different.

**Comment by ehpor** [comment]
Test failures are due to https://github.com/asdf-format/asdf/issues/1172. I'll restart CI once the fix comes out.

**Comment by WilliamJamieson** [comment]
ASDF 2.11.2 and 2.12.1 have been released with jsonschema pins to resolve the issues.

**Review by ehpor — APPROVED** [review]
LGTM.

**Pull request merged**